pulsar-io/elastic-search/src/test/java/org/apache/pulsar/io/elasticsearch/ElasticSearchSinkTests.java

```diff
@@ -177,6 +181,41 @@ public final void send100Test() throws Exception {
         verify(mockRecord, times(100)).ack();
     }
 
+    @Test(enabled = true)
+    public final void sendNoSchemaTest() throws Exception {
```

Comment:
what is this ?

Reply:
it's not directly related to this improvement, I can do another pull. I noticed we didn't have this kind of tests without using the schema